src/main.c

```diff
@@ -199,13 +205,13 @@ void ui_menu_about(void) {
     ux_flow_init(0, ux_menu_about_flow, NULL);
 }
 #else
-static const char *const INFO_TYPES[] = {"Version"};
-static const char *const INFO_CONTENTS[] = {APPVERSION};
+static const char *const INFO_TYPES[] = {"Version", "Developer"};
+static const char *const INFO_CONTENTS[] = {APPVERSION, APPDEVELOPER};
 
 static bool nav_callback(uint8_t page, nbgl_pageContent_t *content) {
     UNUSED(page);
     content->type = INFOS_LIST;
-    content->infosList.nbInfos = 1;
+    content->infosList.nbInfos = 2;
     content->infosList.infoTypes = (const char **)INFO_TYPES;
     content->infosList.infoContents = (const char **)INFO_CONTENTS;
```

Comment:
```suggestion
    content->infosList.infoTypes = INFO_TYPES;
    content->infosList.infoContents = INFO_CONTENTS;
```
cast no needed anymore